maintainers/maintainer-list.nix

```diff
@@ -4199,6 +4199,11 @@
     githubId = 104795;
     name = "Marek Mahut";
   };
+  mmai = {
+    email = "henri.bourcereau@gmail.com";
+    github = "mmai";
```

Comment:
Since https://github.com/NixOS/nixpkgs/pull/66361 you need to provide the githubId, see comment at the top of this file

Reply:
fixed